packages/zowe-explorer/src/job/actions.ts

```diff
@@ -285,12 +285,36 @@ export async function deleteCommand(job: IZoweJobTreeNode, jobsProvider: IZoweTr
         return;
     }
 
-    // delete selected nodes
-    if (nodes.length > 0) {
-        for (const node of nodes) {
-            await jobsProvider.delete(node);
-            deletedNodes.push(`${node.job.jobname}(${node.job.jobid})`);
-        }
+    // delete selected multiple nodes
+    if (nodes.length > 1) {
+        await vscode.window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: localize("deleteJobPrompt.deleteCounter", "Deleting nodes"),
+                cancellable: true,
+            },
+            async (progress, token) => {
+                const total = 100;
+                for (const [index, currNode] of nodes.entries()) {
+                    if (token.isCancellationRequested) {
```

Comment:
Minor: just for your awareness, there is a much nicer (for me personally) [way](https://github.com/microsoft/vscode-extension-samples/blob/main/progress-sample/src/extension.ts) to handle the cancellation requests in the VSCode progress bars. 

It will provide the logical separation between the incrementing and cancellation of the long running action.